Library/Homebrew/utils/copy.rb

```diff
@@ -0,0 +1,58 @@
+# typed: true
```

Comment:
```suggestion
# typed: strict
```

might be nice just because I think you're most (maybe all) of the way there already.

Reply:
Adding `sig` to `Copy#extra_flags` would cause the following error:

```
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/sorbet-runtime-0.5.11391/lib/types/private/class_utils.rb:115:in `block in replace_method': You're trying to replace `extra_flags` on `#<Class:Utils::Copy>`, but that method exists in a prepended module (#<Class:0x000000010e826d60>::MacOSOverride), which we don't currently support. (RuntimeError)
```

(Which is why I didn't add `sig` to the method in the first place.)

`Utils::Bottles#tag` has `sig` and it is overridden seemingly without the same issue, so I guess there is a way around it. But I have no idea of that.

Reply:
Ok, thanks @tesaguri, makes sense!